avm/res/databricks/workspace/main.bicep

```diff
@@ -116,6 +116,33 @@ param accessConnectorResourceId string = ''
 @description('Optional. The default catalog configuration for the Databricks workspace.')
```

Comment:
> Adding a comment here, this may not be working as expected. idempotency failing on key vault error. Looking into it

Just blocking on your behalf @clintgrove . Only comments to files are considered merge-blocking comments.